cmd/redirector/main.go

```diff
@@ -67,7 +67,7 @@ func redirect(w http.ResponseWriter, req *http.Request) {
 	referer := req.Header.Get(hdrReferer)
 
 	// TODO(ahmetb): remove once https://github.community/t/chrome-85-breaks-referer/130039 is fixed
-	if referer == "https://github.com/" {
+	if referer == "https://github.com/" && repoParam != "" {
```

Comment:
Shouldn't it be repoParam=="" ?

If there's the repo parameter specified, we don't want to show the form?

Reply:
Eeek. Yeah, nice catch.  I had tested this but failed to test it correctly.